q2_demux/_summarize/assets/overview.html

```diff
@@ -25,9 +25,28 @@ <h1>Demultiplexed sequence counts summary</h1>
 </div>
 {% endif %}
 
+<div class="row">
```

Comment:
Can this table get moved into the Quality Plots tab? You can probably get away with just sticking this in as a new row near the bottom of the page (maybe?)